collector/README.md

```diff
@@ -384,23 +392,29 @@ The mandatory `<PROFILER>` argument must be one of the following.
   - **Output**. .dot and .txt file (.txt likely is what you want to see first).
   - **Notes**. Works primarily with incremental compilation kinds.
 
-The mandatory `<RUSTC>` argument is a patch to a rustc executable, similar to
-`bench_local`.
+The mandatory `<RUSTC>` argument is a path to a rustc executable or a
+`+`-prefixed toolchain specifier, the same as for `bench_local`.
 
-The mandatory `<ID>` argument is an identifer that will form part of the
-output filenames.
+The identifier that forms part of the output filenames is chosen in a similar
+fashion to the one chosen for `bench_local`.
 
 ### Profiling options
 
 The following options alter the behaviour of the `profile_local` subcommand.
 - `--builds <BUILDS>`: as for `bench_local`.
 - `--cargo <CARGO>`: as for `bench_local`.
 - `--exclude <EXCLUDE>`: as for `bench_local`.
+- `--id <ID>`: an identifer that will form part of the output filenames.
+  Also, if `--rustc2` is identified, a `1` will be appended to the ID for the
+  first run and a `2` will be appended to the ID for the second run.
```

Comment:
Do you think it's worth clarifying that this appending only happens with `--id` or path-based rustc's, not with the +-based synthesis?

As I think about this, it may make sense to actually change that to appending `-1` or `-2` if the ID would otherwise be the same before and after. This would let us easily e.g. do trivial comparisons for noise by passing identical toolchains before/after. It might be easier to explain, too.

Reply:
It's worth clarifying, yes.

I don't want to append `-1` and `-2` if the toolchains are the same; that's an additional edge case and thus makes it harder to explain. I think I'll error out in that case. If anyone wants to test the same toolchain twice they can use `--id` and get different IDs that way.